development.cfg

```diff
@@ -2,3 +2,9 @@
 extends =
     test-plone-4.3.x.cfg
     https://raw.github.com/4teamwork/ftw-buildouts/master/plone-development.cfg
+
+find-links +=
+    http://psc.4teamwork.ch/simple
```

Comment:
Done... And yes we have. 

egov.contactdirectory... if you install it with the contact extra. We may fix this later...